src/client/entity-editor/author-credit-editor/author-credit-section.tsx

```diff
@@ -91,7 +96,7 @@ function AuthorCreditSection({
 	const authorCreditPreview = _map(authorCreditEditor, (credit) => `${credit.name}${credit.joinPhrase}`).join('');
 	const authorCreditRows = _values(authorCreditEditor);
 
-	const isValid = validateAuthorCreditSection(authorCreditRows, authorCreditEnable);
```

Comment:
is this check necessary do this check even if author credit are unabled at first place ? 🤔